### ISSUE TITLE
github download url could possibly be templated

### ISSUE THREAD

**Issue by fgeorgatos** [hpcugent/easybuild-framework#1814]
fi. `https://github.com/git/git/archive` should work out of the box

`source_urls = ['https://github.com/alexdobin/STAR/archive/'` may need `github_org=alexdobin`
similarly, https://github.com/EddyRivasLab/hmmer/tree/3.0 may need `github_org=EddyRivasLab`
(thinking of it, non-default capitalization is an issue here, maybe insert a `github-repo` combo string)

fyi. this is spin-off from https://github.com/hpcugent/easybuild-easyconfigs/issues/3242

**Comment by boegel** [comment]
```
# https://github.com/%(github_account)s/%(name)s/archive
source_urls = [GITHUB_SOURCE]
# or via GitHub API?
github_account = 'foo'
```

**Comment by fgeorgatos** [comment]
yes, that would work! and looks good. 

idea: also templating `github_commitid` might help where upstream does a poor job in tagging a version;
as a bonus, you get decent certainty of downloading what you really meant (remember Rgputools?)

**Comment by boegel** [comment]
fixed in #1872
